apps/storybook.namekit.io/stories/Namekit/TextArea.stories.tsx

```diff
@@ -92,11 +91,21 @@ export const CustomClass: Story = {
   },
 };
 
-export const Disabled: Story = {
+export const PrimaryDisabled: Story = {
   args: {
     variant: "primary",
     size: "medium",
-    value: "Disabled Textarea",
+    value: "Primary Disabled Textarea",
+    rows: 4,
+    disabled: true,
+  },
+};
+
+export const SecondaryDisabled: Story = {
+  args: {
+    variant: "secondary",
+    size: "medium",
+    value: "Secondary Disabled Textarea",
```

Comment:
Please see the issues I reported about using `value` like this.

Reply:
Fixed